raiden/tests/scenarios/bf1_basic_functionality.yaml

```diff
@@ -222,11 +222,13 @@ scenario:
             - assert_events:
                 contract_name: "TokenNetwork"
                 event_name: "ChannelClosed"
-                num_events: 1
+                num_events: 0
                 event_args: {closing_participant: 0}
```

Comment:
Here we want to assert that _no_ `ChannelClosed` event happened, which is nice. :+1:

Reply:
I'd flip the order of the two events though, that way we are sure that the channel was settled without an actual `close` step in between. what do you think?

Reply:
makes sense